examples/gnrc_networking/Makefile

```diff
@@ -37,6 +37,9 @@ USEMODULE += netstats_rpl
 # USEMODULE += sock_dns              # include DNS client
 # USEMODULE += gnrc_ipv6_nib_dns     # include RDNSS option handling
 
+# Enable handling of the Route Information Option
+USEMODULE += gnrc_ipv6_nib_rio
```

Comment:
I'm fine with dropping this here and only including it in the `gnrc_networking-subnets` example